content/download/weekly/index.adoc

```diff
@@ -0,0 +1,49 @@
+---
+layout: simplepage
+title: Weekly Release Line
+---
+:toc:
+
+The Jenkins _Weekly Release Line_ delivers bug fixes and new features rapidly to users and plugin developers who need them.
+These releases are generally delivered on a weekly cadence, hence the name.
+For more conservative users, there is also a 
+link:/download/lts/[Long-Term Support (LTS)] line.
+
+## Model
+
+The "weekly" release cadence is **aspirational**.
+Generally we target releasing Jenkins every week, but sometimes we do out-of-order releases.
+To get notifications about new releases, it is possible to subscribe to link:https://www.jenkins.io/changelog/rss.xml[releases RSS feed].
```

Comment:
or github releases?

Reply:
Right now we do not have automation for them, so I would not document it until the process is fully automated and GitHub Releases are included into our release Pipeline

Reply:
it's better than RSS imo even if not fully automated but yeah would be great to finish that off